projects/packages/connection/src/class-rest-connector.php

```diff
@@ -129,6 +129,15 @@ public function __construct( Manager $connection ) {
 					'registration_nonce' => array(
 						'description' => 'The registration nonce',
 						'type'        => 'string',
+						'required'    => true,
+					),
+					'no_iframe'          => array(
+						'description' => 'Disable In-Place connection flow and go straight to Calypso',
+						'type'        => 'boolean',
+					),
+					'redirect_uri'       => array(
+						'description' => 'URI of the admin page where the user should be redirected after connection flow',
```

Comment:
Do we want those things to be translatable, here and above?

Reply:
Good catch, fixed!